scripts-available/CDB_TableUtils.sql

```diff
@@ -62,6 +61,13 @@ BEGIN
       );
     END IF;
 
+    -- Disable trigger for tracking updates to run manually
+    EXECUTE FORMAT(
+      'ALTER TABLE %I.%I DISABLE TRIGGER track_updates',
+      schema_name,
+      dest_table_name
+    );
+
     -- Truncate table
     EXECUTE FORMAT('TRUNCATE TABLE %I.%I', schema_name, dest_table_name);
 
```

Comment:
Good observation - they probably should be consistent.  I will however have to verify if this change will work for all uses of this function in cartodb.

Note that not specifying a schema is not the same thing as specifying the `public` namespace.  Table resolution looks for the table in all schemas in the `search_path` configuration parameter, in order, and chooses the first. You may just need to set `search_path` to the correct value for your use case.

Reply:
Spoke with Tyler.  Although search_path will work he will change the truncate/insert to explicitly use the schema to match the drop/replace.  This will remove the perquisite for callers to set proper search_paths.